R/utils-plot.R

```diff
@@ -98,23 +98,44 @@ DelineateYearBoundaries <- function(object, years) {
 
 #' Extend y-Axis Limits
 #'
-#' @description Extends the Y-Axis limits if the error bars extend further than the current limits
+#' @description Extends the Y-Axis limits if the error bars extend further than the current limits.
+#' Sets the y-limits to the smallest (and largest) value between the error bars and current plot limits.
 #' @param gsplot The gsplot object to extend limits for
-#' @param error_bar_args the error bar arguments to check the limits against
-extendYaxisLimits <- function(gsplot, error_bar_args){
-  side <- ifelse(!is.null(error_bar_args[['side']]), error_bar_args[['side']], 2)
+#' @param limits The minimum and maximum points to compare to Y-axis limits
+#' @param side The axis to be changed/compared to (default is 2, the y-axis).
+extendYaxisLimits <- function(gsplot, limits, side){
```

Comment:
can we name the limits argument, `newLimits` or something? To make it more clear. Also, in the description, be sure to say that is is a list of length 2, where names are minPoint and maxPoint. Or make it simpler by just having it be a vector of form `c(minimum, maximum)`. Then update `getErrorBarYLims` to return limits as a vector of length 2